mlflow_databricks_artifacts/protos/databricks.proto

```diff
@@ -0,0 +1,287 @@
+syntax = "proto2";
```

Comment:
Because the proto definitions for ACLed artifacts APIs are not available in older versions of MLflow, we include `databricks_artifacts.proto` and its dependencies (`databricks.proto` / `scalapb.proto`)